src/services/organizeImports.ts

```diff
@@ -13,37 +13,50 @@ namespace ts.OrganizeImports {
         host: LanguageServiceHost,
         program: Program,
         preferences: UserPreferences,
-        skipDestructiveCodeActions?: boolean
+        mode = OrganizeImportsMode.All,

```

Comment:
Nit: Instead of making this default value make it requred and set "All" at call site.